aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -32,28 +31,21 @@ private void Awake()
 
         _playerActions.Move.performed += inputContext => _horizontalInput = inputContext.ReadValue<Vector2>();
         _playerActions.Jump.performed += _ => _movement.OnJumpPressed();
-        _playerActions.LookX.performed += inputContext => _mouseInput.x = inputContext.ReadValue<float>();  
-        _playerActions.LookY.performed += inputContext => _mouseInput.y = inputContext.ReadValue<float>();
-        _uiActions.ShowMouse.performed += _ => _mouseLook.OnShowMousePressed();
-        _uiActions.Click.performed += _ => _mouseLook.OnLeftMousePressed();
+        _uiActions.ShowMouse.performed += _ => _mouseLock.OnShowMousePressed();
+        _uiActions.Click.performed += _ => _mouseLock.OnLeftMousePressed();
     }
 
+    /// <summary>
+    /// Get the change in mouse position since the last frame.
+    /// </summary>
+    public Vector2 GetMouseDelta() => _playerActions.Look.ReadValue<Vector2>();
+
```

Comment:
If you want to do this in the same style as with the movement update, you would put a ReceiveMouseInput(...) function in the cinemachine script or the camera manager and then call that in the update of this Inputmanager. 
It doesn't really matter, but why did you do it this way?

Reply:
this is a clean oneline solution designated specifically to an input manager, as that manages the input and probably should not yet again receive it from somewhere else